starknet_py/contract.py

```diff
@@ -392,6 +396,7 @@ async def invoke(
         :param auto_estimate: Use automatic fee estimation (not recommended, as it may lead to high costs).
         :param nonce: Nonce of the transaction.
         :param tip: The tip amount to be added to the transaction fee.
+        :param auto_estimate_tip: Use automatic fee estimation. Using this option may lead to higher costs.
```

Comment:
Please replace it in other occurrences as well:
```suggestion
        :param auto_estimate_tip: Use automatic tip estimation. Using this option may lead to higher costs.
```